src/App.css

```diff
@@ -2,7 +2,6 @@
 @media (max-width: 768px) {
   /* Mobile logo styles - significantly increased size */
   .logo-container {
-      position: sticky;
```

Comment:
This fixes the issue where the logo was overlapping some of the page content (namely the `input-container` div).